src/rules/useStrictRule.ts

```diff
@@ -34,61 +35,51 @@ class UseStrictWalker extends Lint.ScopeAwareRuleWalker<{}> {
         return {};
     }
 
-    public visitModuleDeclaration(node: TypeScript.ModuleDeclarationSyntax): void {
-        var childCount = TypeScript.childCount(node.modifiers);
+    public visitModuleDeclaration(node: ts.ModuleDeclaration) {
+        var modifiers = node.modifiers;
+        var hasDeclareModifier = (modifiers != null) && (modifiers.length > 0) && (modifiers[0].kind === ts.SyntaxKind.DeclareKeyword);
 
         // current depth is 2: global scope and the scope created by this module
-        // but skip declare module statements
-        if (this.getCurrentDepth() === 2 &&
-            !(childCount > 0 && TypeScript.childAt(node.modifiers, 0).kind() === TypeScript.SyntaxKind.DeclareKeyword)) {
-
-            if (this.hasOption(UseStrictWalker.OPTION_CHECK_MODULE)) {
-                this.checkUseStrict(node, node.moduleElements);
+        if (this.getCurrentDepth() === 2 && !hasDeclareModifier) {
+            if (node.body != null && this.hasOption(UseStrictWalker.OPTION_CHECK_MODULE)) {
+                this.handleBlock(node, <ts.Block> node.body);
```

Comment:
Right, I commented on the thing down below. Although now that I think about it, is there a reason why you can't use the union type in the handleBlock now that we have it, since we're using 1.4?